Source/Docs/demos.rst

```diff
@@ -7,30 +7,21 @@ MOSA includes three demos applications to demostrate various capabilities of the
 
 .. tip:: You can start any demo by launching its respective script in the Demos folder.
 
-HelloWorld
-==========
-
-The **HelloWorld** demo is the classic rendition of `"Hello, World!" <https://en.wikipedia.org/wiki/%22Hello,_World!%22_program>`__, except it does not actually display those words. Instead it displays basic information hardware information, such as the BIOS information, the CMOS time, and the Multiboot memory map.
-
-.. image:: images/mosa-demo-helloworld.png
-
-TestWorld
+GraphicalWorld
 =========
 
-The **TestWorld** demo performs version simple, basic tests to validate the compiler, runtime, and kernel functionality.
+The **GraphicalWorld** demo uses, as its name would suggest, the GPU and some other features offered by MOSA to make a nice, graphical demo.
 
-.. image:: images/mosa-demo-testworld.png
+.. image:: images/mosa-demo-graphicalworld.png
 
-CoolWorld
-=========
+HelloWorld
+==========
 
-The **CoolWorld** demo utilitizes the MOSA device drivers system to enumerate hardware devices and instantiate drivers, such as for the IDE controller, disk devices, and FAT file system.
+The **HelloWorld** demo is the classic rendition of `"Hello, World!" <https://en.wikipedia.org/wiki/%22Hello,_World!%22_program>`__, except it does not actually display those words. Instead it implements a basic shell with a few basic commands, like boot information, memory usage, and device information.
 
-.. image:: images/mosa-demo-coolworld.png
+.. image:: images/mosa-demo-helloworld.png
 
-SVGAWorld
+TestWorld
 =========
 
-The **SVGAWorld** demo uses the VMware SVGA II driver for graphics, and uses multiple other features (similar to the ones used by **CoolWorld**) offered by MOSA to make a nice demo.
-
-.. image:: images/mosa-demo-svgaworld.webp
+The **TestWorld** demo is a barebones demo meant to show how you can create a barebones kernel using MOSA.
```

Comment:
Let's discuss demo name. Also, barebones is minimalistic wording. Let's use our mission statement: "As operating systems share a lot of common groundwork, MOSA aims to standardize interfaces in order to foster portability and to provide both projects with basic implementations of these interfaces." to guide as.